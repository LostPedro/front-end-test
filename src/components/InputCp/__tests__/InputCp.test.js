import React from 'react'
import {render, cleanup} from '@testing-library/react'
import {InputCp} from '../'

afterEach(cleanup)

it('should take a snapshot', () => {
  const {asFragment} = render(<InputCp />)

  expect(asFragment(<InputCp />)).toMatchSnapshot()
})
