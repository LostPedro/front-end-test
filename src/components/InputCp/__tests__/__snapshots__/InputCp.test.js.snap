// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="input-component"
  >
    <label
      class=""
      for=""
    />
    <input
      class="input-component-input"
      id=""
      name=""
      required=""
      type=""
      value=""
    />
  </div>
</DocumentFragment>
`;
